### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: doc/DISCLAIMER.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+**Warning:** The package can take **15 to 30 minutes** to complete depending on your system configuration. **Don't intrupt the installation process while installing.**
+
+## Important points to read before installing
+1. **Plume** is still in **pre-release** stage and undergoing heavy development, so there can be **bugs**.
+1. **Plume** require a dedicated **root domain**, eg. plume.domain.tld
+1. **Plume** require a valid **certificate** installed on the domain. Yunohost can **install Letsencrypt certificate** on the domain from **admin web-interface** or through **command-line**.
```
Comment (tituspijean) [inline doc/DISCLAIMER.md]:
```suggestion
1. **Plume** requires a dedicated **root domain**, eg. plume.domain.tld
1. **Plume** requires a valid **certificate** installed on the domain. YunoHost can **install a Let's Encrypt certificate** on the domain from **admin web-interface** or through **command-line**.
```

File: manifest.json [hunk 2]
```diff
@@ -56,7 +60,7 @@
                 "type": "boolean",
                 "ask": {
                     "en": "Is registrations open to everyone?",
-                    "fr": "Les inscriptions sont t'elles ouvertes à tout le monde?"
+                    "fr": "Les inscriptions sont t'elles ouvertes à tout le monde ?"
```
Comment (tituspijean) [inline manifest.json]:
```suggestion
                    "fr": "Les inscriptions sont-elles ouvertes à tout le monde ?"
```

**Review by tituspijean — APPROVED** [review]
Code LGTM, besides some typos. ;)

File: scripts/install [hunk 3]
```diff
@@ -33,6 +32,7 @@ instance_name=$YNH_APP_ARG_NAME
 registration=$YNH_APP_ARG_REGISTRATION
 admin_email=$(ynh_user_get_info $admin 'mail')
 secret_key=$(ynh_string_random --length=32 | base64)
+architecture=$YNH_ARCH
```
Comment (yalh76) [inline scripts/install:35]: From where `$YNH_ARCH` is initialised ?

File: scripts/install [hunk 4]
```diff
@@ -98,13 +97,10 @@ ynh_system_user_create --username=$app --home_dir="$final_path"
 #=================================================
 ynh_script_progression --message="Creating a PostgreSQL database..."
 
-ynh_psql_test_if_first_run
 db_name=$(ynh_sanitize_dbid --db_name=$app)
-db_user=$db_name
-db_pwd=$(ynh_string_random --length=30)
 ynh_app_setting_set --app=$app --key=db_name --value=$db_name
-ynh_app_setting_set --app=$app --key=db_pwd --value=$db_pwd
-ynh_psql_setup_db --db_user=$db_user --db_name=$db_name --db_pwd=$db_pwd
```
Comment (yalh76) [inline scripts/install]: Why removing `db_user` and `db_pwd` ?
It makes ugly sentences like `--db_user=$db_name` or using archaic `psqlpwd`

**Review by yalh76 — CHANGES_REQUESTED** [review]

File: scripts/install (hunk 3 above)
Comment (ericgaspar) [reply scripts/install:35]: `$YNH_ARCH` replaces `ynh_detect_arch` helper
https://github.com/YunoHost/yunohost/blob/fa163cbc2e44a5c54ac7d48adff7c573da6c2b30/debian/changelog#L93

https://github.com/YunoHost/yunohost/blob/fa163cbc2e44a5c54ac7d48adff7c573da6c2b30/src/yunohost/app.py#L2393

Comment (yalh76) [reply scripts/install:35]: Ok cool, would be great to have some information about that in the documentation ... I updated to Yunohost version requirement

**Comment by yalh76** [comment]
!testme

**Review by alexAubin — APPROVED** [review]

**Pull request merged**